graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -144,7 +144,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name of the GraphiQL template parameters.
          */
-        public static final String TEMPLATE_PARAMETERS = PREFIX + ".template-parameters";
+        public static final String TEMPLATE_PARAMETERS_CONFIG = PREFIX + ".template-parameters";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.